libraries/I2S/src/pio_i2s.pio

```diff
@@ -50,17 +50,17 @@ right:
 
 ;                           +----- WCLK
 ;                           |+---- BCLK
-    mov x, y         side 0b11
-left:
-    out pins, 1      side 0b00
-    jmp x--, left    side 0b01
-    out pins, 1      side 0b00
-
     mov x, y         side 0b01
-right:
+left:
     out pins, 1      side 0b10
-    jmp x--, right   side 0b11
+    jmp x--, left    side 0b11
```

Comment:
Don't you need to swap the WCLK polarity throughout the program, too?  The label changes don't affect anything in the generated code and now, I'd assume, L and R are swapped?

Reply:
The diff is misleading (see the lines number), I didn't change the labels but inverted the WCLK values only.

Reply:
Got it, thx.  I see the shifted SIDESETs later on.